api/google_api/util/SceGoogleApiHandler.js

```diff
@@ -178,6 +178,107 @@ class SceGoogleApiHandler {
       });
     });
   }
+
+  /**
+   * Adds an event to a given calendar by its id.
+   * @param calendarId {string} calendar id for which calendar to pull from
+   * @param newEvent {Event} event to translate and add to Google Calendar
+   */
+  addEventToCalendar(calendarId, newEvent) {
+    return new Promise((resolve, reject) => {
+      const calendar =
+        google.calendar({ version: 'v3', auth: this.oAuth2Client });
+      let eventToAdd = this.translateEvent(newEvent);
+      calendar.freebusy.query({
+        resource: {
+          timeMin: eventToAdd.start.dateTime,
+          timeMax: eventToAdd.end.dateTime,
+          timeZone: eventToAdd.start.timeZone,
+          items: [{ id: 'primary' }],
+        },
+      },
+      (err, res) => {
+        if(err) resolve(err);
+        const eventsArr = res.data.calendars.primary.busy;
+        if(eventsArr.length === 0) {
+          const response = calendar.events.insert({
+            auth: this.oAuth2Client,
+            calendarId: calendarId,
+            resource: eventToAdd
+          });
+          resolve(response.status);
+        }
+        resolve('Event Conflict!');
+      });
+    });
+  }
+
+  /**
+   * Convert SCE events to Google Calendar Events
+   * @param {Object} eventToAdd An SCE Event
+   * @returns {Object} A Google Calendar formatted event
+   */
+  translateEvent(eventToAdd) {
+    let event = {
+      'summary': eventToAdd.title,
+      'location': eventToAdd.eventLocation,
+      'description': eventToAdd.description,
+      'start': {
+        'dateTime': this.dash(eventToAdd.eventDate) + 'T'
+        + this.to24(eventToAdd.startTime) + ':00-07:00',
+        'timeZone': 'America/Los_Angeles'
+      },
+      'end': {
+        'dateTime': this.dash(eventToAdd.eventDate) + 'T'
+        + this.to24(eventToAdd.endTime) + ':00-07:00',
+        'timeZone': 'America/Los_Angeles'
+      },
+      'recurrence': [
+        'RRULE:FREQ=DAILY;COUNT=1'
+      ],
+      'attendees': [],
+      'reminders': {
+        'useDefault': false,
+        'overrides': [
+          {'method': 'email', 'minutes': 24 * 60},
+          {'method': 'popup', 'minutes': 10}
+        ]
+      }
+    };
+    return event;
+  }
+
+  /**
+   * Convert date in slash format to a date in dash format
+   * @param {string} date in slash format
```

Comment:
give an example of what the date looks like originally and how its formatted
```suggestion
   * @param {string} date in slash format <example>
   * @returns {string} <example formatted date>
```